crm_compassion/models/event_compassion.py

```diff
@@ -151,6 +153,12 @@ class event_compassion(models.Model):
     parent_copy = fields.Many2one(
         'account.analytic.account', related='parent_id')
     calendar_event_id = fields.Many2one('calendar.event')
+    hold_start_date = fields.Date(
+        'Hold Start Date',
+        required=True,
+        default=lambda self: int(self.env['ir.config_parameter'].get_param(

```

Comment:
This is wrong because you assign an integer to a date field. Instead of using this default attribute I propose you to add a `@api.onchange('start_date')` method, so that the field is set when the start_date is selected.

see https://www.odoo.com/documentation/8.0/reference/orm.html#module-openerp.api